src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.stars__body {
+  margin: 0;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. The `.stars__body` class seems to be styling a `body` tag, which is not recommended. Consider using a different class name that does not imply styling a tag name directly.